docs/datascience/jupyter-notebooks.md

```diff
@@ -278,22 +278,38 @@ If you want to use the full set of debugging features supported in VS Code, such
 
 ![Debug cell button](images/jupyter/debug-cell.png)
 
+### Search through notebook
+
+You can search (Ctrl / Cmd + F) through an entire notebook or you can search parts of your notebook by filtering your search options. When searching through your notebook, click on the Filter option (funnel icon) to search across:
+- Markdown cell input (`Markdown Source`)
```

Comment:
Use bold for UI text so **Markdown Source**, **Rendered Markdown**, **Code Cell Source**, and **Cell Output**